.github/workflows/hash-backend-cd.yml

```
@@ -11,7 +11,7 @@ on:
       - "libs/@local/hash-isomorphic-utils/**"
       # TODO: remove/rewrite these as we remove the package directory.
       - "apps/hash-graph/**"
-      - "packages/hash/**"
+      - "apps/hash/**"
```

Something is off here. I guess we mean 

```suggestion
      - "apps/hash-*/**"
```

because `apps/hash` only contains a single `README.md`. Overall, I wonder if we an place that file somewhere else, e.g. turning it into a section of `apps/README.md`. This way we won't have to have `apps/hash`, which is currently a folder amongst other Yarn workspaces that is not a workspace itself.